app/components/Dashboard/BestNewMusicTab/BestNewMusicMenu/BestNewMusicCard/index.js

```diff
@@ -24,24 +30,94 @@ function toFavoriteTrack({ artist, title, thumbnail }) {
   };
 }
 
+function getReleaseInfo({ artist, title }) {
```

Comment:
I didn't want it to be a part of `redux` action as I didn't see any data to be stored.